src/main/webapp/webpack.config.dev.js

```diff
@@ -8,7 +8,7 @@ exports.config = {
       target: "localhost:8080"
     },
     overlay: {
-      warnings: true,
+      warnings: false,
```

Comment:
Any specific reason to suppress webpacker warnings?

Reply:
So all warning were showing up and it was slowing development.  They still get sent to the console so you will see them there.